index.bs

```diff
@@ -4361,11 +4361,11 @@ following algorithm.</p>
        direction">horizontal</a></dt>
        <dd>
         <dl class="switch">
-         <dt>If the <a>WebVTT cue line alignment</a> is <a lt="WebVTT cue position center
```

Comment:
Yeah, though I'd like to change the internal terms, this is way too error prone. Bikeshed has better ways to do this... :-)